src/routes/Main/Settings/About.tsx

```diff
@@ -43,6 +43,10 @@ export default () => {
           <ExternalLinkIcon mx="2px" />
         </Link>
       </Text>
+      <Text fontSize={"sm"}>
+        This is an beta release, meaning the software may have bugs and
+        unforeseen problems.
+      </Text>
```

Comment:
In beta, Timetabl is supposed to be more stable. The beta status is not really due to there being many bugs, but only due to it being incomplete. This is why I removed the message after the beta release. If you wish you could modify this message so that it indicates this, or you could close the PR.

Reply:
@debater-coder you can edit this